### DISCUSSION TITLE
Negotiation of EthereumStratum/2.0.0 failed

### DISCUSSION THREAD

**Discussion by glennsheary** [catchthatrabbit/coreminer#48] [Miner]
is anyone able to explain what's going on here?
it was working for me yesterday.

coreminer 0.19.0+commit.0607dbad
Build: linux/release/gnu

cp 12:11:29 main     RandomY hard aes mode is enabled
cp 12:11:29 main     RandomY full mem mode is enabled
cp 12:11:29 main     RandomY JIT mode is enabled
 i 12:11:29 main     Configured pool eu.catchthatrabbit.com:8008
 i 12:11:29 <unknown> Selected pool eu.catchthatrabbit.com:8008
 i 12:11:30 <unknown> Negotiation of EthereumStratum/2.0.0 failed. Trying another ...
 X 12:11:30 <unknown> Error  86.110.248.193:8008 [ Operation canceled ]
 X 12:11:30 <unknown> No more IP addresses to try for host: eu.catchthatrabbit.com
 X 12:11:30 <unknown> No more IP addresses to try for host: eu.catchthatrabbit.com
 X 12:11:30 <unknown> No more IP addresses to try for host: eu.catchthatrabbit.com
 i 12:11:30 <unknown> Disconnected from eu.catchthatrabbit.com:8008
 i 12:11:30 <unknown> No connection. Suspend mining ...
 i 12:11:30 <unknown> No more connections to try. Exiting...
 i 12:11:30 <unknown> Got interrupt ...
 i 12:11:30 main     Terminated!

**Comment by rastislavcore** [comment]
WIP. That isn't an issue for now, but we are fixing it.
Please, ignore it till the next version excludes it.

**Comment by hmercker** [comment]
This issue persists. Cannot get connection with EU pool. Any fixes possible?

**Reply by rastislavcore** [reply]
Please, consider opening the [issue](https://github.com/catchthatrabbit/coreminer/issues/new/choose) when the problem is essential.
We are fixing the messages now.

**Reply by hmercker** [reply]
all good, thanks
